.craft.yml

```diff
@@ -67,4 +67,5 @@ targets:
           - nodejs10.x
           - nodejs12.x
           - nodejs14.x
+          - nodejs16.x
```

Comment:
AWS Lambda doesn't support Node.js v16 yet: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
Sad panda.

Reply:
At least GH Actions does and we're green there.